brainstormforce/ultimate-addons-for-gutenberg
# Fix: Block condition conflicting with Lesson navigation block and formidable forms

### Description

- Trello - https://trello.com/c/55pzPhTV/254-block-condition-conflicting-with-lesson-navigation-block-and-formidable-forms

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install LifterLMS and Formadale forms
- Create formidable forms and drag and drop on pages and check is it visible in an editor or not.
- Go to LifterLMS > Lessons and courses. Create new lessons and courses.
- And the check is the Lesson and Courses block visible in an editor or not.

### Checklist:
